moveit_ros/hybrid_planning/local_planner/local_constraint_solver_plugins/src/forward_trajectory.cpp

```diff
@@ -60,10 +61,15 @@ bool ForwardTrajectory::initialize(const rclcpp::Node::SharedPtr& node,
   {
     stop_before_collision_ = node->declare_parameter<bool>("stop_before_collision", false);
   }
-  planning_scene_monitor_ = planning_scene_monitor;
   node_ = node;
   path_invalidation_event_send_ = false;
   num_iterations_stuck_ = 0;
+
+  planning_scene_monitor_ = planning_scene_monitor;
```

Comment:
@AndyZe I re-enabled passing the a configured planning scene monitor down from the local_planning component